spec/openapi.yaml

```diff
@@ -626,135 +645,190 @@ components:
 
     FileInfo:
       type: object
+      required:
+        - name
+        - path
+        - ro
+        - type
+        - c_timestamp
+        - m_timestamp
```

Comment:
I'm not sure we have both `c_timestamp` and `m_timestamp` available (might need to check that out). Why are both mandatory?

Reply:
Mostly, what I'm trying to say, can you pick only one that's mandatory?